pilot/pkg/model/service.go

```diff
@@ -56,12 +56,19 @@ type Service struct {
 	// Address specifies the service IPv4 address of the load balancer
 	Address string `json:"address,omitempty"`
 
-	// Protect concurrent ClusterVIPs read/write
+	// Protect concurrent read/write
 	Mutex sync.RWMutex
```

Comment:
Should not use the same lock for different fields to improve the possibility of contention.